### PR TITLE
Nostr: auto generate nostr secret

### PR DESCRIPTION
**Pull request by abhay-raizada**

### CODE REVIEW

**Review by Nodirbek75 — APPROVED** [review]

File: app/hooks/use-nostr-profile.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      try {
+        console.log("Looking for nostr creds in keychain")
+        const credentials = await Keychain.getInternetCredentials(KEYCHAIN_NOSTRCREDS_KEY)
+        if (credentials) {
+          console.log("nostr creds found in keychain", credentials)
+          setNostrSecretKey(credentials.password)
+          return
+        }
+        const nostrSecret = nip19.nsecEncode(generateSecretKey())
+        console.log("Generated new nostr secret:", nostrSecret)
```
Comment (islandbitcoin) [inline app/hooks/use-nostr-profile.ts:20]: need to remove console.logs before merging to main

File: app/hooks/use-nostr-profile.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const KEYCHAIN_NOSTRCREDS_KEY = "nostr_creds_key"
+  const [nostrSecretKey, setNostrSecretKey] = useState<string>("")
+
+  useEffect(() => {
+    const initializeNostrProfile = async () => {
+      try {
+        console.log("Looking for nostr creds in keychain")
+        const credentials = await Keychain.getInternetCredentials(KEYCHAIN_NOSTRCREDS_KEY)
+        if (credentials) {
+          console.log("nostr creds found in keychain", credentials)
```
Comment (islandbitcoin) [inline app/hooks/use-nostr-profile.ts:15]: remove console.log showing nsec credentials

**Review by islandbitcoin — CHANGES_REQUESTED** [review]
Remove logs showing nsec

File: app/hooks/use-nostr-profile.ts (hunk 1 above)
Comment (abhay-raizada) [reply app/hooks/use-nostr-profile.ts]: done

**Review by islandbitcoin — APPROVED** [review]
ack

**Pull request merged**